paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreenState.kt

```diff
@@ -1,11 +1,11 @@
 package com.stripe.android.link.ui.updatecard
 
 import androidx.compose.runtime.Immutable
-import com.stripe.android.model.ConsumerPaymentDetails
 import com.stripe.android.paymentsheet.CardUpdateParams
 
 @Immutable
 internal data class UpdateCardScreenState(
-    val paymentDetails: ConsumerPaymentDetails.Card? = null,
+    val paymentDetailsId: String? = null,
```

Comment:
Can we make the `paymentDetailsId` a required field? Feels weird for this to be nullable.